contracts/Proofs.sol

```diff
@@ -20,15 +19,15 @@ contract Proofs {
     downtime = __downtime;
   }
 
-  mapping(ProofId => bool) private ids;
-  mapping(ProofId => uint256) private starts;
-  mapping(EndId => uint256) private ends;
-  mapping(ProofId => EndId) private idEnds;
-  mapping(ProofId => uint256) private probabilities;
-  mapping(ProofId => uint256) private markers;
-  mapping(ProofId => uint256) private missed;
-  mapping(ProofId => mapping(uint256 => bool)) private received;
-  mapping(ProofId => mapping(uint256 => bool)) private missing;
+  mapping(SlotId => bool) private ids;
+  mapping(SlotId => uint256) private starts;
+  mapping(RequestId => uint256) private ends;
+  mapping(SlotId => RequestId) private requestIds;
```

Comment:
Nitpick personal preference, other naming options:
- `slotRequests`
- `slotRequestIds` 
- `requestsForSlot`
It was confusing for me to read `requestIds[slotId]`, but at the end of the day, it's not a problem as the compiler will enforce the right types. More of a personal preference.

Reply:
Agreed. I'll change it to `requestForSlot`.